dlp/src/main/java/com/example/dlp/QuickStart.java

```diff
@@ -56,40 +60,49 @@ public static void main(String[] args) throws Exception {
     // instantiate a client
     try (DlpServiceClient dlpServiceClient = DlpServiceClient.create()) {
```

Comment:
The info type names (`US_MALE_NAME` and `US_FEMALE_NAME`) above may need to be updated (use `PERSON_NAME` instead - but it encompasses both of these).

Reply:
Done.